guide/spanish/miscellaneous/storing-mongo-username--password-persistently-using-dotenv/index.md

```diff
@@ -69,7 +69,7 @@ Esta solución mantiene su código limpio de las credenciales seguras que no des
 
 [stackoverflow.com](http://stackoverflow.com/questions/35356692/best-practice-when-using-an-api-key-in-node-js)
 
-[! \[Drake Main](https://i.stack.imgur.com/jRaTj.jpg?s=128&g=1) \] (http://stackoverflow.com/users/4956243/drake-main) #### [Práctica recomendada al usar una clave API en Node.js](http://stackoverflow.com/questions/35356692/best-practice-when-using-an-api-key-in-node-js)
+[! \[Drake Main](https://cdn-media-1.freecodecamp.org/imgr/jRaTj.jpg) \] (http://stackoverflow.com/users/4956243/drake-main) #### [Práctica recomendada al usar una clave API en Node.js](http://stackoverflow.com/questions/35356692/best-practice-when-using-an-api-key-in-node-js)
```

Comment:
Format of the image is broken. Image not showing up

Reply:
The issue might  be pre existing, because the image itself loads.
https://cdn-media-1.freecodecamp.org/imgr/jRaTj.jpg
![Drag Racing](https://cdn-media-1.freecodecamp.org/imgr/jRaTj.jpg)

In order to do a proper search and replace for all of the similar instances in different languages, I would like to differ this to another PR.

Reply:
Alright, I guess you can open an issue and we can take this up there